nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/ExecuteScriptMarkLogic.java

```diff
@@ -161,7 +178,7 @@ public ValidationResult validate(String subject, String input, ValidationContext
         .description("Receives the original FlowFile if the call to MarkLogic fails for any reason").build();
 
 
-    private static Charset UTF8 = StandardCharsets.UTF_8;
+    private static Charset UTF8 = Charset.forName("UTF-8");
```

Comment:
This is undoing some changes from #170 - please make sure the PR only has changes needed for DEVEXP-62.